src/Admin/Block/Imagine/SlideshowBlockAdmin.php

```diff
@@ -58,32 +57,28 @@ protected function configureListFields(ListMapper $listMapper)
      */
     protected function configureFormFields(FormMapper $formMapper)
     {
-        parent::configureFormFields($formMapper);
+        if (null === $this->getParentFieldDescription()) {
+            parent::configureFormFields($formMapper);
+        }
 
         $formMapper
-            ->with('form.group_general')
-                ->add('title', TextType::class, array('required' => false))
-            ->end()
-            ->with('Items')
-                ->add('children', CollectionType::class,
-                    array(),
-                    array(
-                        'edit' => 'inline',
-                        'inline' => 'table',
-                        'admin_code' => $this->embeddedAdminCode,
-                        'sortable' => 'position',
-                    ))
+            ->tab('form.tab_general')
+                ->with('form.group_block', null === $this->getParentFieldDescription()
```

Comment:
is this ever null? we call parent above, so i would expect this to always have something.